### ISSUE TITLE
Python class and parameter colors are white

### ISSUE THREAD

**Issue by T3nt4c135** [Jaredk3nt/laserwave#38]
![Capture](https://user-images.githubusercontent.com/35921587/132084026-a8c447ac-7adc-4158-9f62-32300db240da.PNG)
What it looks like on most other skins
![color](https://user-images.githubusercontent.com/35921587/132084145-8dc282a3-97fd-47f2-9b47-963a1bd71c3f.PNG)

**Comment by Jaredk3nt** [comment]
@T3nt4c135 What is that other theme you are using? As far as I can see there isn't a specific scope that can be used to target the imported class. All of the default vscode themes are showing it remaining as the base color to me

![image](https://user-images.githubusercontent.com/6362515/132778983-1b572f92-a778-4f16-9bed-c8885f6c6bfe.png)

Default dark:
![image](https://user-images.githubusercontent.com/6362515/132779130-801d4433-d8d6-4d34-9d79-91d53dc78935.png)

**Comment by T3nt4c135** [comment]
@Jaredk3nt The theme posted is the official Dracula theme. This reminds me why I switched out of the default theme to begin with. Python is interesting in that way.

**Comment by Jaredk3nt** [comment]
Yeah I agree I much prefer the colored class as well, let me see if I can figure out how Dracula does it any try to fit that into Laserwave

**Comment by Jaredk3nt** [comment]
![image](https://user-images.githubusercontent.com/6362515/134281683-fef81cf7-d0ac-4f93-8a6a-38ca2151b5b6.png)

Hmm... even on Dracula I'm not seeing the same thing :( very odd.
I'm not sure how to proceed with that one. I can't get a scope for it, I'm curious if its something wrong with the most recent version of VSCode? I will keep my eye out for it and see if I can find a solution but can't promise anything at this point.

I did however fix a couple of other things I found while checking it out, they are in v1.3.1 of LaserWave

Old:
![image](https://user-images.githubusercontent.com/6362515/134281762-a2287fa9-f6e1-42b4-8c82-e61071187718.png)
New:
![image](https://user-images.githubusercontent.com/6362515/134281794-fb0c62e2-29eb-45ef-bcd9-e4836ab139dd.png)
